lib/wicked_pdf/wicked_pdf_helper/assets.rb

```diff
@@ -231,12 +231,11 @@ def running_in_development?
       end
 
       def webpacker_version
-        return unless defined?(Webpacker)
-
-        # If webpacker is used, need to check for version
-        require 'webpacker/version'
-
-        Webpacker::VERSION
+        if defined?(Shakapacker)
+          Shakapacker::VERSION
+        elsif defined?(Webpacker)
+          Webpacker::VERSION
```

Comment:
Did you run your app locally with this fix? Aren’t there other places where the Webpacker constant is being referenced?  Curious if this would fix the initial error but make way for another.

Reply:
Hi!

Yes I monkey patched it in my app, and it is working properly, running in production.

As I have stated in my PR description above:
- The issue is caused by this line: `require 'webpacker/version'`
- The issue could be resolved by just removing this line. (`webpacker/version` would be present anyway, no need to require it again.)
- To clarify once more, you will not get an error when referencing `Webpacker` in code, but you will get one when you try to require `webpacker/version` .

This is happening because Shakapacker decided to slowly rename everything "webpacker" to "shakapacker" in their codebase. At the moment they are trying to provide backwards compatibility.
- This backwards compatibility is the reason why referencing `Webpacker` doesn't raise an error. When using `shakapacker` version `7` .
- But since they already renamed much of their source files, you can not directly require `webpacker/version`, because no such file exists. You would need to require `shakapacker/version` under `shakapacker` version `7` .
  - This is present in the project: https://github.com/shakacode/shakapacker/blob/v7.0.2/lib/shakapacker/version.rb
  - This is not: https://github.com/shakacode/shakapacker/blob/v6.6.0/lib/webpacker/version.rb

If you do not like the idea of removing that require statement, then I can change my fix to the following:
```ruby
def webpacker_version
  if defined?(Shakapacker)
    require 'shakapacker/version'          # add new require statement for shakapacker version 7+
    Shakapacker::VERSION
  elsif defined?(Webpacker)
    require 'webpacker/version'            # keep the old require statement
    Webpacker::VERSION
  end
end
```

To clarify once more, wicked_pdf aside from this one line of code ( `require 'webpacker/version'` ) is `shakapacker` version `7` compatible. If we remove or change this line in the above mentioned way, then we end up with full compatibility.

This full compatibility is the goal of my current PR.

Reply:
@unixmonkey @AJFaraday I'm not sure how this was tested, but this is not working for us. We currently have version 2.6.3, and the change to 2.7.0 fails with an uninitialized constant Webpacker::VERSION error.

The same happens when I load the rails console and try to access the constant.

Note that nowhere in the webpacker code I see the require of the version file.

Reply:
@sk- I would be happy to take a fix PR.